packages/mina-service/src/service/transaction-service.ts

```diff
@@ -0,0 +1,105 @@
+import { ETransactionStatus } from '@zkdb/common';
+import { MinaNetwork } from '@zkdb/smart-contract';
+import { ModelTransaction } from '@zkdb/storage';
+import { schedule } from 'node-cron';
+
+// Base on Mina protocol blockscan we divide to 10
+const CRON_SCHEDULE = '*/2 * * * *'; // Cron expression to run every 30 minutes
```

Comment:
is 30min too long? this means users must wait at most 30min for the transaction status to be updated

Reply:
it's 2min but I forgot to change the comment, let me check

Reply:
Also consider making sure that you account for data races. For example, what happens if the task runs longer than 2 minutes?
- Do you allow overlapping scheduled runs? If yes:
- What happens when two overlapping runs acquire the same transaction and try to update it?

Reply:
I've just added the isRunning to check